src/sentry/charts/chartcuterie.py

```diff
@@ -60,26 +60,23 @@ def generate_chart(self, style: ChartType, data: Any, upload: bool = True) -> Un
             "data": data,
         }
 
-        with Session() as session:
```

Comment:
I couldn't get the traces to connect with session.reques even when I manually added the sentry-trace header

Reply:
double checking @EvanPurkhiser was there any specific reason to use sessions here instead of requests?

Reply:
I think we're actually generally trying to use our Session over requests, but let's just go with this for now 👍